src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -31,7 +32,10 @@
      [react/text {:style           styles/profile-user-name
                   :font           :medium
                   :number-of-lines 1}
-      name])])
+      name])
+   [react/text {:style           styles/profile-three-words
```

Comment:
can we add condition `(not= name (gfy/generate-gfy public-key))` to do not show 3w name twice

Reply:
makes sense

Reply:
@flexsurfer done!